examples/computed-field-key/computed_field_key.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/io;
+
+const ID = "unique-identifier";
+
+function getDetailMap(string uniqueId,
+                      string firstName,
+                      string lastName,
+                      string idName, 
+                      string id) returns map<string> {
+    return {
+        // The value `"unique-identifier"` is subtituted from the constant as the key.
```

Comment:
```suggestion
        // The `unique-identifier`value is substituted from the constant as the key.
```

Reply:
Will include quotes to reflect the value.

```suggestion
        // The `"unique-identifier"` value is substituted from the constant as the key.
```